netlify/functions/feedback-management.mts

```diff
@@ -18,12 +27,30 @@ const getSlackChannelId = (url, specialCases) => {
   if (isDesignFeedback) {
     return DESIGN_FEEDBACKS_CHANNEL_ID;
   }
+
   if (url.includes('/x/')) {
+    if (url.includes('/x/react-charts')) {
+      return X_CHARTS_FEEBACKS_CHANNEL_ID;
+    }
+    if (url.includes('/x/react-date-pickers') || url.includes('/x/react-tree-view')) {
+      return X_EXPLORE_FEEBACKS_CHANNEL_ID;
+    }
+    if (url.includes('/x/react-data-grid')) {
```

Comment:
This doesn't work, e.g. https://mui.com/x/api/data-grid/data-grid/. Is there a way we could rely on productId rather than URLs? For example, once we move to subdomains, this will break.